qiskit_experiments/library/tomography/tomography_experiment.py

```diff
@@ -102,59 +137,58 @@ def __init__(
 
         # Measurement basis and qubits
         self._meas_circ_basis = measurement_basis
-        if measurement_qubits:
+        if measurement_indices:
             # Convert logical qubits to physical qubits
-            self._meas_qubits = tuple(measurement_qubits)
-            self._meas_physical_qubits = tuple(self.physical_qubits[i] for i in self._meas_qubits)
-            for qubit in self._meas_qubits:
+            self._meas_indices = tuple(measurement_indices)
+            self._meas_physical_qubits = tuple(self.physical_qubits[i] for i in self._meas_indices)
+            for qubit in self._meas_indices:
                 if qubit not in range(self.num_qubits):
                     raise QiskitError(
                         f"measurement qubit ({qubit}) is outside the range"
                         f" of circuit qubits [0, {self.num_qubits})."
                     )
         elif measurement_basis:
-            self._meas_qubits = tuple(range(self.num_qubits))
+            self._meas_indices = tuple(range(self.num_qubits))
             self._meas_physical_qubits = self.physical_qubits
         else:
-            self._meas_qubits = tuple()
+            self._meas_indices = tuple()
```

Comment:
Is this not following the docs "If None all circuit physical qubits will be measured.", or is that done somewhere else?

Same comment for preparations.

Reply:
The behavior in the base tomography experiment depends on if `measurement_basis` kwarg is used (its default is None unlike in QPT and QST). If the basis is not None, the case above this one sets it to all the qubits, if it is None then this clause makes it so measurement is performed (eg measurement tomo with only preparations). Similarly for `preparation_basis` (so you can do QST, instead of QPT). I should probably add more detail to the `measurement_basis` and `preparation_basis` kwargs.